dogs.html

```diff
@@ -0,0 +1,193 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Page</title>
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Neuton:ital@0;1&family=Work+Sans:ital,wght@0,300;0,600;1,300&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./styles/normalize.css" />
+    <link rel="stylesheet" type="text/css" href="./styles/styles.css">
+  </head>
+  <body>
+    <a href="#main-content" class="hidden">Skip to main content</a>
+    <main>
+      <header class="flex-header">
+        <div class="flex-header-1">
+          <img src="./images/adoptadog-logo.svg" alt="Adoptadog logo">
+          <h2 class="header-headline">Seattle's premiere dog adoption agency</h2>
+        </div>
+        <div class="flex-header-2">
+          <div class="flex-cart">
+            <img class="flex-icon" src="./images/cart-icon.svg" alt="shopping cart icon">
+            <p class="flex-price">$1234.56</p>
+          </div>
+          <a href="./checkout.html">
+            <p class="green-button">Checkout</p>
+          </a>
+        </div>
+      </header>
+      <nav>
+        <ul class="flex-nav">
+          <li class="flex-nav-home"><a href="./index.html">Home &nbsp; &nbsp;</a></li>
+          <li class="flex-nav-dogs"><a href="./dogs.html">Dogs &nbsp; &nbsp;</a></li>
+          <li class="flex-nav-blog"><a href="./blog.html">Blog</a></li>
+        </ul>
+      </nav>
+      <section class="main-container">
+        <div class="grid-main-heading">
+          <h1>Here are all of our Dogs!</h1>
+        </div>
+        <div class="grid-main3">
```

Comment:
You are on the right track here. I think you might be able to simplify some of this. Good call making a container to hold all of your dog tiles. Would suggest using grid-template on your grid. You can see a good example of this here: [https://css-tricks.com/snippets/css/complete-guide-grid/](https://css-tricks.com/snippets/css/complete-guide-grid/)